enterprise-suite/scripts/pull-console-api.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
```

Comment:
Should add `set -eu` ideally.

Reply:
I'll add `set -u`. The way I'm checking for previous command exit status seems incompatible with `set -e`, the script then exits immediately without printing the reason it failed.

Reply:
Right - you need to use `set +e` around the section where you want to manually handle exit code checking.

Reply:
But that is basically the whole file in this case, so I'll leave it without `set -e`